backend.md

```diff
@@ -1,75 +1,74 @@
+2. The `coverageArea` field follows the `GeoJSON MultiPolygon` format (https://en.wikipedia.org/wiki/GeoJSON);
+3. The `document` must be a unique field;
+4. The `id` must be a unique field, but not necessarily an integer;
 
+You can use this [json](files/pdvs.json) file composed of hundreds of partner information to test your application.
+Below, you can have a look at how these partners might be represented in a map:
 ![Partners in map](files/images/pdvs.png)
 
-Before talking about the challenge itself, let me give you some information about the partners' specification:
-We have some requirements that you must pay attention:
-
-1. The `address` is defined by the format `GeoJSON Point` (https://en.wikipedia.org/wiki/GeoJSON)
-2. The `coverageArea` is defined by the format `GeoJSON MultiPolygon` (https://en.wikipedia.org/wiki/GeoJSON) 
-3. The `document` must be a unique field (as it is the CNPJ)
-
-## What we want you to do
-
-Considering all the info we gave to you, we expect you to develop a service that provides API using REST or GraphQL and enable the following functionalities. The programming language and the database you choose to use it is entirely up to you. 
-
-### 1. Create partner: 
-
-All fields are required and must follow the rules set above.
-
-### 2. Get partner by id:
-
-Get a specific partner by its `id`.
-
-### 3. Search partner:
-
-Given a specific location (coordinates `lng` and `lat`), search the nearest partner considering each partner's coverage area.
+### 1.2. Load partner by id:
+Return a specific partner by its `id` with all the fields presented above.
 
-### 4. Tech Requirements, Docs and Deploy:
+### 1.3. Search partner:
+Given a specific location (coordinates `long` and `lat`), search the **nearest** partner **which the coverage area includes** the location.
 
-* Your project must be **cross-platform**.
-* Provide a documentation so we know how to execute your service locally and how to deploy it (*focus on simplicity, and don't forget that we should test your service on our own, without further assistance*)
+### 1.4. Technical Requirements:
+* The programming language and the database engine are entirely up to you;
+* Your project must be **cross-platform**;
+* Provide a documentation (README.md) file explaining how to execute your service **locally** and how to deploy it (*focus on simplicity, and don't forget that we should test your service on our own, without further assistance*).
 
 ## Evaluation Method
 
-Your code will be under review of the Zx's Global Engineering team. What we will look for:
-- **Performance**
-- **Testing**
-- **Maintanability**
-- **Separation of concerns**
-- **Software engineering**
+We will evaluate your code challenge based on some [system quality attributes](https://en.wikipedia.org/wiki/List_of_system_quality_attributes).
+Some we consider a must-have, like **correctness**, and will be evaluated on a boolean (works/follows or not) approach.
```

Comment:
```suggestion
Some we consider a must-have, like **correctness**, and will be evaluated on a binary (works/follows or not) approach.
```

Reply:
I was looking for this! My head could only remember of bool